numpyro/infer/hmc_gibbs.py

```diff
@@ -358,7 +359,7 @@ def body_fn(i, val):
     return gibbs_fn
 
 
-def subsample_gibbs_fn(model, model_args=(), model_kwargs={}):
+def subsample_gibbs_fn(model, model_args=(), model_kwargs={}, block_updates={}):
```

Comment:
Could you add `, *,` before this new argument (like `discrete_gibbs_fn`)? I don't have a strong opinion about `block_updates` but would something like `block_sizes` or `num_blocks` deliver a better context?

Reply:
I'll make the update. Agree that `num_blocks` is a better name.